tests/system/test_cluster/test_agent_groups/test_agent_groups_forced_change.py

```diff
@@ -0,0 +1,180 @@
+'''
```

Comment:
I can't see where you connect the agent with the master or with the worker according to cases 1 and 2. We need to do a sync.